dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -24,6 +24,8 @@ che:
     - '/etc/passwd:/etc/passwd:ro'
 <% end -%>
   ports:
+    - '32001:32001'
```

Comment:
what are these ports ? JMX ?
I would say they should be exposed only if a property is defined